eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ledger.hs

```diff
@@ -295,19 +294,22 @@ data ConwayLedgerEvent era
   = UtxowEvent (Event (EraRule "UTXOW" era))
   | CertsEvent (Event (EraRule "CERTS" era))
   | GovEvent (Event (EraRule "GOV" era))
+  | MempoolEvent (Event (EraRule "MEMPOOL" era))
```

Comment:
Could this be  `MempoolEvent Text` instead ? Don't know if the wrapping brings anything useful

Reply:
Per our discussion: better no